tests/Hedgehog.Tests/MinimalTests.fs

```diff
@@ -65,11 +63,10 @@ let rec genExp : Gen<Exp> =
         ]
 
         Gen.choiceRec recs nonrecs
-        |> Gen.shrink shrinkExp // comment this out to see the property fail
+        |> Gen.shrink shrinkExp
     )
 
-[<Fact>]
-let ``greedy traversal with a predicate yields the perfect minimal shrink``() =
+let ``greedy traversal with a predicate yields the perfect minimal shrink`` () =
```

Comment:
Since we're exposing this for `Hedgehog.Benchmarks`, I'd prefer a simple name without double backticks.

Reply:
I renamed it to perfectMinimalSrink.